### ISSUE TITLE
Question about training dataset

### ISSUE THREAD

**Issue by CrossEntropy** [xucong-zhang/ETH-XGaze#35]
Hi, Zhang. Thanks for ur project! My issue is why did u use original normalized image in training loop? In other words, why didn't u use any image enhancement? like random gray, random color distortion ....
Looking forward for ur reply! :)
